src/main/scala/cromwell/binding/types/WdlArrayType.scala

```diff
@@ -27,3 +27,14 @@ case class WdlArrayType(memberType: WdlType) extends WdlType {
     case _ => false
   }
 }
+
+object WdlArrayType {
+
+  implicit class WdlArrayEnhanced(wdlType: WdlType) extends WdlType {
+
+    override protected def coercion: PartialFunction[Any, WdlValue] = wdlType.coercion
```

Comment:
It doesn't, but because it extends WdlType it needs to implement `coercion` and `toWdlString`, so we reuse the already existing implementation.
The added value is the `isAnArrayOf` method a few lines below which removes some boilerplate.